homeassistant/components/youtube/sensor.py

```diff
@@ -25,6 +25,8 @@
 )
 from .entity import YouTubeChannelEntity
 
+PARALLEL_UPDATES = 1
```

Comment:
How does this apply to sensors?

Reply:
I actually did this since the platinum quality scale required it and I saw `airly` also having a DataUpdateCoordinator and having this set. Up on reading the documentation of this again, is this only needed for entities that have update functions like `async_update` or `_handle_coordinator_update`?

Reply:
Well this is used no to update multiple sensors at the same time to prevent hammering an API, but since this is using a data update coordinator, the data from this sensor doesn't origin from an API, but from the coordinator data...

So there is not much use for it?